radio/radio_test.ts

```diff
@@ -138,6 +138,20 @@ describe('<md-radio>', () => {
       expect(a2.element.checked).withContext('prev radio checked').toBeFalse();
     });
 
+    it('Using arrow right should fire a change event', async () => {
```

Comment:
nit: `'dispatched a change event on user navigation'` since it's not explicitly with right arrows only